components/producers/producer.go

```diff
@@ -113,28 +117,28 @@ func ParseMultiJSONMessages(in []byte) ([]interface{}, error) {
 // WriteDraconOut provides a generic method to write the resulting protobuf to the output file.
 func WriteDraconOut(
 	toolName string,
-	issues []*v1.Issue,
+	issues []*draconapiv1.Issue,
 ) error {
 	source := getSource()
-	cleanIssues := []*v1.Issue{}
+	cleanIssues := []*draconapiv1.Issue{}
 	for _, iss := range issues {
 		iss.Description = strings.ReplaceAll(iss.Description, sourceDir, ".")
 		iss.Title = strings.ReplaceAll(iss.Title, sourceDir, ".")
 		iss.Target = strings.ReplaceAll(iss.Target, sourceDir, ".")
 		iss.Source = source
 		cleanIssues = append(cleanIssues, iss)
-		log.Printf("found issue: %+v\n", iss)
+		slog.Info(fmt.Sprintf("found issue: %+v\n", iss))
 	}
-	scanStartTime := strings.TrimSpace(os.Getenv(EnvDraconStartTime))
+	scanStartTime := strings.TrimSpace(os.Getenv(components.EnvDraconStartTime))
 	if scanStartTime == "" {
 		scanStartTime = time.Now().UTC().Format(time.RFC3339)
 	}
-	scanUUUID := strings.TrimSpace(os.Getenv(EnvDraconScanID))
-	scanTagsStr := strings.TrimSpace(os.Getenv(EnvDraconScanTags))
+	scanUUUID := strings.TrimSpace(os.Getenv(components.EnvDraconScanID))
+	scanTagsStr := strings.TrimSpace(os.Getenv(components.EnvDraconScanTags))
 	scanTags := map[string]string{}
 	err := json.Unmarshal([]byte(scanTagsStr), &scanTags)
 	if err != nil {
-		log.Printf("scan with uuid %s does not have any tags, err:%s", scanUUUID, err)
+		slog.Info(fmt.Sprintf("scan with uuid %s does not have any tags, err:%s", scanUUUID, err))
```

Comment:
This doesn't look like a correct use of the logger. the default logger setup will add the uuid as a key value pair in the log entry. Also this should be an invocation of `Error` instead of `Info`